### DISCUSSION TITLE
mpc play and stop crons on Raspbian OS

### DISCUSSION THREAD

**Discussion by dinesh-kumar-c** [MusicPlayerDaemon/MPD#1389] [Q&A] [answered]
Hi Community members - Good day.

I am trying to start and stop music from cron using mpc stop and start from user 'pi' , but I get  following error from cron. Please advise. I tried user 'pi' to 'audio' , but still the issue exists. Thanks
```
mpd error: you don't have permission for "stop"
```

my MPD process starts as a system process from systemd directory /lib/systemd/system/mdp.service. the /etc/mpd.conf is as below

```
music_directory		"/var/lib/mpd/music"

playlist_directory		"/var/lib/mpd/playlists"

db_file			"/var/lib/mpd/tag_cache"

log_file			"/var/log/mpd/mpd.log"

pid_file			"/run/mpd/pid"

state_file			"/var/lib/mpd/state"

```

**Comment by MaxKellermann** [comment, marked as answer]
You configured a password in your configuration file. What you posted is not the configuration file that is actually used by the MPD process you are talking to, or is incomplete.
See https://mpd.readthedocs.io/en/stable/user.html#permissions-and-passwords

**Reply by dinesh-kumar-c** [reply]
Thanks MaxKellermann for pointing to the right resource. I had to export export MPD_HOST=xxxxx@localhost in my cron script, before doing mpc start/stop.
